projects/etos_suite_runner/src/etos_suite_runner/lib/executor.py

```diff
@@ -52,40 +52,32 @@ def __decrypt(self, password):
             return password
         return Fernet(key).decrypt(password_value).decode()
 
-    def __auth(self, username, password, type="basic"):  # pylint:disable=redefined-builtin
+    def __auth(
+        self, username: str, password: str, type: str = "basic"  # pylint:disable=redefined-builtin
```

Comment:
Rename `type` to scheme (as that is the correct naming with regards to http authentication) would remove the need for the pylint:disable.

Reply:
Sadly, this name is defined in the request dictionary that we get from ETOS.